### PR TITLE
Add a great poem and update background color in CSS file.

### PR DESCRIPTION
**Pull request by kbonneteau**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -14,6 +14,11 @@ const poems = [
     poem: "Roses are red \n Violets are blue \n Jerry Seinfeld is considering \n Making Bee Movie 2",
     image: "bee.jpeg",
   },
+  {
+    author: "E. E. Cummings",
```
Comment (ghost) [inline script.js:18]: hes an okay author.

**Review by ghost — APPROVED** [review]
its' great

**Comment by ghost** [comment]
you are perfect!

**Comment by kbonneteau** [comment]
> you are perfect!

Thank you!

**Pull request merged**